Mathlib/CategoryTheory/Limits/ConcreteCategory.lean

```diff
@@ -95,7 +95,9 @@ theorem Concrete.multiequalizer_ext {I : MulticospanIndex.{w} C} [HasMultiequali
   apply Concrete.limit_ext
   rintro (a | b)
   · apply h
-  · rw [← limit.w I.multicospan (WalkingMulticospan.Hom.fst b), comp_apply, comp_apply, h]
+  · rw [← limit.w I.multicospan (WalkingMulticospan.Hom.fst b), comp_apply, comp_apply]
+    erw [h]
+    rfl
```

Comment:
```suggestion
  · rw [← limit.w I.multicospan (WalkingMulticospan.Hom.fst b), comp_apply, comp_apply]
    dsimp at *
    rw [h]
```
suffices here. (I think it's nicer like this, as we can then see that our `dsimp` lemmas are achieving a good form, whereas with an `erw` we could be doing an arbitrarily hard defeq problem.)